events/companies/models.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from registration.models import User
+
+
+class Company(models.Model):
+    name = models.CharField(max_length=50, null=False)
+    description = models.TextField(max_length=500, null=True, blank=True)
+    admin = models.OneToOneField(User)
+
+    def __str__(self):
+        return '{}'.format(self.name)
+
+    @staticmethod
+    def get_all():
+        return Company.objects.all()
+
+    @staticmethod
+    def get_by_id(company_id):
+        try:
+            return Company.objects.get(pk=company_id)
+        except Company.DoesNotExist:
+            return None
+
+    @staticmethod
+    def get_teams(company_id):
+        return [team.name for team in Company.get_by_id(company_id).teams.all()]
+
+    @staticmethod
+    def get_user_company(request):
+        company = Company.objects.filter(admin=request.user).first()
```

Comment:
@VKotsovskyi Can't really say. Would the `try-except` match better for this purpose?